app/Providers/AppServiceProvider.php

```diff
@@ -80,6 +81,10 @@ public function register()
         if ($this->app->environment(['local', 'develop'])) {
             $this->app->register(\Laravel\Dusk\DuskServiceProvider::class);
         } 
+
+        $this->app->singleton('ArieTimmerman\Laravel\SCIMServer\SCIMConfig', SnipeSCIMConfig::class); //do we need this? yes.
```

Comment:
Please explain why we need this if you are going to leave the comment in (actually even if you remove the comment, add one that explains it.) If you had to ask yourself this, someone else (probably me) will ask it later.

Reply:
Totally fair ask.